tests/functional/extension_module_test.php

```diff
@@ -40,14 +40,17 @@ static public function tearDownAfterClass(): void
 
 	protected function tearDown(): void
 	{
-		$this->uninstall_ext('foo/bar');
+		if (self::$tests_count == 1)
```

Comment:
I think the variable name `$tests_count` is too ambiguous to determine if extensions need to be uninstalled

Reply:
It's to determine if the last test in the queue has been executed (1 means exactly that), and then extensions get uninstalled.
What name would suite for that purpose?

Reply:
Can you perhaps use `tearDownAfterClass()` for this use case? https://docs.phpunit.de/en/10.5/fixtures.html#sharing-fixture